.github/workflows/ci.yml

```diff
@@ -30,6 +30,19 @@ jobs:
         with:
           node-version: ${{ matrix.node_version }}
 
+      - name: Get yarn cache directory
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Set dependencies cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: v1-dependencies-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
```

Comment:
**question:** Should we swap `runner.os` with `matrix.os`?

Reply:
It is up to the core team. `runner.os` will output `Linux`, `Windows`, and `macOS` while `matrix.os` will output from `matrix.os` list. Since we are using `*-latest`, it is not needed IMO.

Reply:
Yeah okay, your are partly right. But what I originally thought was: should we share cache between linux with node 12, 14 and 15? Or should each instances have their own cache?
I don't know if e.g. node 12 could come in conflict with node 15 :thinking: 

_But thanks for showing me that `runner.*` is a possible variable in GHA_ ^^

Reply:
Thanks for pointing out.
I left to add node version in cache keys.